src/main/resources/paas.yaml

```diff
@@ -1,42 +0,0 @@
-database:
```

Comment:
There's lots of examples of configuration files of a similar shape (although they do different things) already in this repo. I don't think there's much need to keep this around.